eng/tools/tsp-client-tests/test/tsp-client.test.ts

```diff
@@ -0,0 +1,16 @@
+import { execa } from 'execa';
+import { join } from 'path';
+import { test } from 'vitest';
+
+async function tspClient(...args: string[]) {
```

Comment:
Where are the args were going to pass to tsp-client?

Reply:
Does the `convert()` function answer your question?

Reply:
Yes it does :) thanks!